src/graphql/schema.js

```diff
@@ -81,6 +82,7 @@ input ProfileInput {
   avatar: String
   name: String
   bio: String
+  hasStake: Boolean
```

Comment:
Does this need to be added to `ProfileInput`? The server shouldn't rely on the client to provide this.

Reply:
was a way to flag hasStake on users own profile but you're right probably better to not have client handle this